prow/jobs/rafter/rafter.yaml

```
@@ -0,0 +1,88 @@
+job_template: &job_template
+  skip_report: false
+  decorate: true
+  optional: false
+  path_alias: github.com/kyma-project/rafter
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/buildpack-golang-kubebuilder:v20190208-813daef
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
+        args:
+          - "/home/prow/go/src/github.com/kyma-project/rafter"
+        resources:
+          requests:
+            memory: 1.5Gi
+            cpu: 0.8
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-kyma: "true"
+
+presubmits: # runs on PRs
+  kyma-project/rafter:
+    - name: pre-master-rafter
+      always_run: true
+      branches:
+        - ^master$
+      <<: *job_template
+      labels:
+        <<: *job_labels_template
+        preset-build-pr: "true"
+
+postsubmits:
+  kyma-project/rafter:
+    - name: post-master-rafter
+      branches:
+        - ^master$
+      <<: *job_template
+      labels:
+        <<: *job_labels_template
+        preset-build-master: "true"
+        preset-bot-github-token: "true"
+        prow.kyma-project.io/slack.skipReport: "true" # this job will be ignored by Slack reporter
+    - name: post-release-rafter
+      # Prow resolves git tags as branches, so that regexp triggers build after tagging the repository.
+      branches:
+        - 'v\d+\.\d+\.\d+$'
```

It should also match suffixes

```suggestion
        - 'v\d+\.\d+\.\d+(?:-.*)?$'
```